server/views/pages/conditions/select-conditions/index.njk

```diff
@@ -45,95 +45,111 @@
                 items: [
                   {
                     value: "ABI",
-                    text: "ABI" | formatConditionTypeScreenValue | default("ABI", true)
+                    text: "ABI" | formatConditionTypeScreenValue | default("ABI", true),
+                    checked: form.conditions.includes("ABI")
```

Comment:
Added the "checked" property, so that when the form is re-rendered (eg: after an invalid submission) the previously selected values are still checked